src/EnergyPlus/DataHeatBalance.hh

```diff
@@ -1358,13 +1365,14 @@ namespace DataHeatBalance {
     {
         // Members
         bool EnforceZoneMassBalance; // flag to enforce zone air mass conservation
-        bool BalanceMixing;          // flag to allow mixing to be adjusted for zone mass balance
+        int ZoneFlowAdjustment;      // specifies how zone air flow balance is determined (AdjustMixingOnly, AdjustReturnOnly, AdjustMixingThenReturn, AdjustReturnThenMixing, None)
         int InfiltrationTreatment;   // determines how infiltration is treated for zone mass balance
         int InfiltrationZoneType;    // specifies which types of zones allow infiltration to be changed
                                      // Note, unique global object
 
         // Default Constructor
-        ZoneAirMassFlowConservation() : EnforceZoneMassBalance(false), BalanceMixing(false), InfiltrationTreatment(0), InfiltrationZoneType(0)
+        ZoneAirMassFlowConservation()
+            : EnforceZoneMassBalance(false), ZoneFlowAdjustment(1), InfiltrationTreatment(0), InfiltrationZoneType(0)
```

Comment:
Updated the struct using the new member variable that maps to the different options and deleted the existing method. The default method initialization will be corrected to "0".